scene/gui/text_edit.cpp

```diff
@@ -4458,6 +4458,14 @@ bool TextEdit::is_scroll_past_end_of_file_enabled() const {
 	return scroll_past_end_of_file_enabled;
 }
 
+void TextEdit::set_drag_and_drop_selection_enabled(const bool p_enabled) {
+	drag_and_drop_selection_enabled = p_enabled;
+}
+
+bool TextEdit::is_drag_and_drop_selection_enabled() const {
+	return drag_and_drop_selection_enabled;
+}
+
```

Comment:
`.h` and `.c` content should follow the same order, so this should be under `delete_selection` below, same with the other changes. Including moving `drag_and_drop_selection_enabled` under selection.

`Nitpick:` Would move it under `is_deselect_on_focus_loss_enabled`

Reply:
done